composer.json

```diff
@@ -20,7 +21,8 @@
         "symfony/yaml": "~2.7|~3.1",
         "symfony/browser-kit": "~2.7|~3.1",
         "symfony/twig-bundle": "~2.7|~3.1",
-        "symfony/phpunit-bridge": "~2.7|~3.1"
+        "symfony/phpunit-bridge": "~2.7|~3.1",
+        "phpunit/phpunit": "~4.8"
```

Comment:
you should allow both 4.8 and 5.x if you install it as a dependency, because PHPUnit 4.8 does not officially support PHP 7.